components/gitpod-db/go/cost_center.go

```diff
@@ -122,6 +119,60 @@ func (c *CostCenterManager) GetOrCreateCostCenter(ctx context.Context, attributi
 	return result, nil
 }
 
+// computeDefaultSpendingLimit computes the spending limit for a new Organization.
+// If the first joined member has not already granted credits to another org, we grant them the the free credits allowance.
+func (c *CostCenterManager) getSpendingLimitForNewCostCenter(attributionID AttributionID) int32 {
+	_, orgId := attributionID.Values()
+
+	// fetch the first owner that joined the org
+	var userId string
+	db := c.conn.Raw(`
+		SELECT userid
+		FROM d_b_team_membership
+		WHERE
+			teamId = ? AND
+			creationTime = (
+				SELECT min(creationTime)
+				FROM d_b_team_membership
+				WHERE
+					role='owner' AND
+					deleted = 0 AND
+					teamID = ?
+			)
+	`, orgId, orgId).Scan(&userId)
+	if db.Error != nil {
+		log.WithError(db.Error).WithField("attributionId", attributionID).Error("Failed to get userId for org.")
+		return c.cfg.ForTeams
+	}
+
+	if userId == "" {
+		log.WithField("attributionId", attributionID).Error("Failed to get userId for org.")
+		return c.cfg.ForTeams
+	}
+
+	// check if the user has already granted free credits to another org
+	var count int64
+	db = c.conn.Raw(`
```

Comment:
Any reason to prefer raw queries over the ORM syntax?

Reply:
Curious, but also creates some inconcistency. I'd have expected that we resort for raw SQL only for the very complex cases and try to use the ORM for the remainder as it provides some amount of type safety